### PR TITLE
Add option for automatic logging

### PR DESCRIPTION
**Pull request by likeawindrammer**

### CODE REVIEW

File: src/text_file_functions.cpp [hunk 1]
```diff
@@ -1049,3 +1049,9 @@ QString AOApplication::get_casing_can_host_cases()
           .value<QString>();
   return result;
 }
+bool AOApplication::get_casing_auto_logging_enabled()
+{
+    QString result =
+        configini->value("automatic_logging_enabled", "false").value<QString>();
```
[the remark this answers is not in the document]
Comment (oldmud0) [reply src/text_file_functions.cpp:1055]: :+1:, i was going to leave a review saying "why add features if you're going to leave them off by default"

File: src/aooptionsdialog.cpp [hunk 2]
```diff
@@ -723,7 +737,7 @@ void AOOptionsDialog::save_pressed()
   configini->setValue("stickyeffects", ui_stickyeffects_cb->isChecked());
   configini->setValue("stickypres", ui_stickypres_cb->isChecked());
   configini->setValue("customchat", ui_customchat_cb->isChecked());
-
+  configini->setValue("automatic_logging_enabled", ui_casing_log_cb->isChecked());
```
Comment (Crystalwarrior) [inline src/aooptionsdialog.cpp:740]: "ui_casing_log_cb" should be "ui_log_cb"

Comment (likeawindrammer) [reply src/aooptionsdialog.cpp:740]: oops, I missed that one. Will do

**Comment by oldmud0** [comment]
i forgot to merge this, heck

**Pull request merged**